index.html

```diff
@@ -2536,6 +2542,7 @@ <h5>Example Intake Object JSON</h5>
       "biological_sex": "male",
       "gender": "male",
       "phone": "8048675309",
+      "dob": "04/08/1980",
```

Comment:
@rstims While we accept this format (because that's what Amazon was sending us and we needed to patch for it), we prefer ISO-8601 (1980-04-08).

Reply:
@epogue I was wondering about this one